data/tabular/bicerano_dataset/meta.yaml

```diff
@@ -1,95 +1,122 @@
+    units: K
+    type: float
+    names:
+      - noun: computed glass coefficient of thermal expansion at 300K
+  - id: glass_CTE_calc_std
+    description: computed standard deviation of glass coefficient of thermal expansion (CTE) at 300K
+    sampled: false
+    units: K
+    type: float
+    names:
+      - noun: computed standard deviation of glass CTE
+  - id: rubber_CTE_calc
+    description: computed rubber CTE at 300K
+    units: K
+    type: float
+    names:
+      - noun: computed rubber coefficient of thermal expansion (CTE) at 300K
+  - id: rubber_CTE_calc_std
+    description: computed standard deviation of rubber CTE at 300K
+    sampled: false
+    units: K
+    type: float
+    names:
+      - noun: computed standard deviation of rubber coefficient of thermal expansion (CTE) at 300K
 identifiers:
-    - id: PSMILES
-      type: PSMILES
-      description: PSMILES
-    - id: compound_name
-      type: Other
-      names:
-          - noun: compound name
-      description: polymer name
+  - id: PSMILES
+    type: PSMILES
+    description: PSMILES
+  - id: compound_name
+    type: Other
+    names:
+      - noun: compound name
+    description: polymer name
 license: CC BY 4.0
 links:
-    - url: https://pubs.acs.org/doi/10.1021/acsapm.0c00524#
-      description: corresponding publication
-    - url:
-          - https://raw.githubusercontent.com/AdrianM0/chemnlp/main/data/tabular/bicerano_dataset/HT_MD_polymer_properties.csv
-      description: data source
+  - url: https://pubs.acs.org/doi/10.1021/acsapm.0c00524#
+    description: corresponding publication
+  - url:
+      - https://raw.githubusercontent.com/AdrianM0/chemnlp/main/data/tabular/bicerano_dataset/HT_MD_polymer_properties.csv
+    description: data source
 num_points: 315
 bibtex:
-    - |-
-      @article{afzal2021,
-      author = {Afzal, Mohammad Atif Faiz and Browning, Andrea R. and Goldberg, Alexander and Halls, Mathew D. and Gavartin, Jacob L. and Morisato,
-      Tsuguo and Hughes, Thomas F. and Giesen, David J. and Goose, Joseph E.},
-      title = {High-Throughput Molecular Dynamics Simulations and Validation of Thermophysical Properties of Polymers for Various Applications},
-      journal = {ACS Applied Polymer Materials},
-      volume = {3},
-      number = {2},
-      pages = {620-630},
-      year = {2021},
-      doi = {10.1021/acsapm.0c00524}}
+  - |-
+    @article{afzal2021,
+    author = {Afzal, Mohammad Atif Faiz and Browning, Andrea R. and Goldberg, Alexander and Halls, Mathew D. and Gavartin, Jacob L. and Morisato,
+    Tsuguo and Hughes, Thomas F. and Giesen, David J. and Goose, Joseph E.},
+    title = {High-Throughput Molecular Dynamics Simulations and Validation of Thermophysical Properties of Polymers for Various Applications},
+    journal = {ACS Applied Polymer Materials},
+    volume = {3},
+    number = {2},
+    pages = {620-630},
+    year = {2021},
+    doi = {10.1021/acsapm.0c00524}}
+templates:
```

Comment:
A lot of nice prompts! 

I wonder if it makes sense to ask about standard deviations as the model will lack the context of how the simulations have been run. What do you think. 

In addition i'd perhaps replace "molecule" with "polymer" 

Did you also think of trying a prompt of the form "Question: What is a polymer with experimental glass transition temperature of {Tg_exp#} K and experimental density at 300 K of {rho_300K_exp#} g/cc. Answer: A polymer with {PSMILES__description} {PSMILES#}"